hubtools/src/caboose.rs

```diff
@@ -193,10 +224,38 @@ mod tests {
             .board("bar")
             .name("fizz")
             .version("buzz")
+            .epoch("")
             .build();
         assert_eq!(caboose.git_commit(), Ok("foo".as_bytes()));
         assert_eq!(caboose.board(), Ok("bar".as_bytes()));
         assert_eq!(caboose.name(), Ok("fizz".as_bytes()));
         assert_eq!(caboose.version(), Ok("buzz".as_bytes()));
+        assert_eq!(caboose.epoch(), Ok("0".as_bytes()));
+    }
+
+    #[test]
+    fn builder_can_make_caboose_with_non_zero_epoch() {
+        let caboose = CabooseBuilder::default().epoch("1234567890").build();
+        assert_eq!(caboose.epoch(), Ok("1234567890".as_bytes()));
+    }
+
+    #[test]
+    fn builder_will_normalize_empty_epoch() {
```

Comment:
I put this in a few other places, but if the epoch must always be a `u32`, then the builder can just Not Allow the empty string as an argument.

Reply:
Yes that is the right thing to do.